README.md

```diff
@@ -10,17 +10,20 @@ From beginning to end, the Mindwitcher project focuses on utilizing EEG signals
 1. **Spell Casting and Horse calling**: EEG signals related to movement and imagined movement are used to call for the player's horse and cast spells within the game.
```

Comment:
Here, I find it a bit confusing we refer to three functionalities but we list them using only the numbers 1 and 2. Anyone has any other suggestions for this? Can't think of one immediately